CONTRIBUTING.md

```diff
@@ -323,7 +332,64 @@ Update the version's release date and status from the
 [Manage Versions](https://jira.mongodb.org/plugins/servlet/project-config/PHPC/versions)
 page.
 
-### Update version info
+### Trigger the release workflow
+
+Releases are done automatically through a GitHub Action. Visit the corresponding
+[Release New Version](https://github.com/mongodb/mongo-php-driver/actions/workflows/release.yml)
+workflow page to trigger a new build. Select the correct branch (e.g. `v1.18`)
+and trigger a new run using the "Run workflow" button. In the following prompt,
+enter the version number and the corresponding JIRA release number for the 
+release. This will create and push the necessary commits and tag, create a draft
```

Comment:
This would be a good place to link to https://jira.mongodb.org/projects/PHPC and advise that the version ID can be copied from a link in the "Version" column (e.g. "37551" from https://jira.mongodb.org/browse/PHPC/fixforversion/37551 for 1.18.0).

Reply:
Re-worded and included instructions on how to find the ID.